src/lib/stable/did_file/visitor/visit/service.ts

```diff
@@ -41,8 +41,7 @@ export function visitService(
 function serviceToCandidString(
     t: IDL.ServiceClass,
     canisterMethodCandidStrings: string[],
-    initMethodCandidString: string[],
-    postMethodCandidString: string[],
+    initOrPostUpgradeMethodCandidString: string[],
```

Comment:
Should we say `and` instead of `or`, seems like the only place that is used

Reply:
Done